### DISCUSSION TITLE
Which parameters to tweak in order to make an absorption compartment behave as though not present

### DISCUSSION THREAD

**Discussion by PriKalra** [Open-Systems-Pharmacology/Forum#876] [Questions & Problems] [answered]
Dears, 

I would like to shut a part of the absorption compartment in the GI such that it behaves as it is not present and has no influence on the distribution. Which parameters should I change in order to get this effect to work?

Best, 
Pri

**Comment by AndreDlm** [comment, marked as answer]
Hi Pri,
not sure I understand your question correctly, but if you want to disable absorption from a given GI compartment, you could set the `effective surface area enhancement factor` to zero. However, note that your GI compartment will still be present, i.e. your drug has to travel through it before it can enter the next compartment and this can obviously have an effect on e.g. t<sub>max</sub> and other PK parameters.
If you want to completely remove a GI compartment, I would try setting organ and liquid volumes, blood flows, surface area enhancement factors etc. to zero. If this won't work, I would directly go to MoBi to cut it out...
Best,
André
